recipes/bimg/all/test_v1_package/CMakeLists.txt

```
@@ -0,0 +1,12 @@
+cmake_minimum_required(VERSION 3.8)
+
+project(test_package LANGUAGES CXX)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup(TARGETS)
+
+find_package(bimg REQUIRED CONFIG)
+
+add_executable(${PROJECT_NAME} test_package.cpp)
+set_target_properties(${PROJECT_NAME} PROPERTIES CXX_STANDARD 14 CXX_STANDARD_REQUIRED ON CXX_EXTENSIONS OFF)
+target_link_libraries(${PROJECT_NAME} bimg::bimg)
```

```suggestion
add_subdirectory(${CMAKE_CURRENT_SOURCE_DIR}/../test_package/
                 ${CMAKE_CURRENT_BINARY_DIR}/test_package/)
```
You can avoid duplication by loading ../test_package